src/NuGet.Core/NuGet.Commands/CommandRunners/PackageSearchRunner.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol;
+using NuGet.Commands.Internal;
+
+namespace NuGet.Commands.CommandRunners
+{
+    public static class PackageSearchRunner
```

Comment:
That's a good point. At first, I had it in the NuGet.CommandLine.XPlat, however, I thought, if I have it in the NuGet.Command project it would allow the `nuget.exe search` command re use the code. As a result, I was able to re use the code by moving it to NuGet.COmmand. However, after the design spec meeting, we agreed it would be a better output to have a table with search term highlighted in red, rather than just listing the packages. As of now, it does not re use the code because the two have two different output designs. If we agree to have exactly the same output design for both,  we could easily call the `PackageSearchRunner` from the NuGet.CommandLine search command.